src/Profile.cpp

```diff
@@ -572,4 +617,9 @@ namespace geopm
     {
         return m_tprof_table;
     }
+
+    void ProfileImp::enable_pmpi(void)
+    {
+
+    }
```

Comment:
Is this supposed to be empty?

Reply:
yes, one reason why I don't love this workaround.  Only DefaultProfile implements any behavior